swap_meet/item.py

```diff
@@ -1,2 +1,28 @@
+import uuid
 class Item:
-    pass
\ No newline at end of file
+
+    def __init__(self, id=None, condition= 0.0): # initializing constructor 
+        self.id = id if id is not None else int(uuid.uuid4()) # creating an id attribute, getting random ID
+        self.condition = condition # setting a condition attribute, a float
+
+    def get_category(self):
+        return f"{self.__class__.__name__}" #returning a string to name each class. 
+    
+    def __str__(self):
+        return f"An object of type Item with id {self.id}." #setting a string to convey the items ID
+    
+    def condition_description(self): # setting the condition data structure. 
+        
+        condition_dict = {
+            0: "Horrible",
+            1: "Damaged",
+            2: "Average",
+            3: "Good",
+            4: "Great",
+            5: "Mint"
+        }
```

Comment:
This is really nice! I love how you all noticed that this was more concise than chained `if` statements! I'm very impressed with this code! This tells me that you all are learning when to use `dictionaries`, you noticed the direct relationship between the two values.